dateparser/date.py

```diff
@@ -39,6 +39,7 @@
 RE_NBSP = re.compile(u'\xa0', flags=re.UNICODE)
 RE_SPACES = re.compile(r'\s+')
 RE_TRIM_SPACES = re.compile(r'^\s+(\S.*?)\s+$')
+RE_TRIM_COLONS = re.compile(r'(\S.*?):*$')
```

Comment:
If we only strip from the beginning, I think `\s*:+$` and `.sub(r'', date_string)` make a bit more sense: we don’t find a match unless there is at least one colon, and we also remove any extra spacing there might be before the colon (I assume space after the colon would have be stripped already somewhere else, but I did not check).

Reply:
> I assume space after the colon would have be stripped already somewhere else, but I did not check

It wouldn't, and it's probably a bug... `'31/07/2019: '` won't be parsed, because `RE_TRIM_SPACES` assumes spaces present at both sides of the date string.
I'm not sure if it should be handled in this PR though.

Reply:
not sure what's the best approach here. what do you think?

Reply:
I'd leave it for another PR. Your approach undeniably fixes the target issue - and that's the point.